.github/workflows/test_interface_portal.yml

```diff
@@ -54,11 +60,26 @@ jobs:
 
       - name: 'Test: Code'
         working-directory: ${{ env.interfacePath }}
-        run: 'npm run test:ci'
+        run: 'npm run test:coverage'
+
+      - uses: qltysh/qlty-action/coverage@v1
+        with:
+          oidc: true
+          add-prefix: interfaces/portal/
+          tag: portal
+          files: ${{ env.interfacePath }}/coverage/lcov.info
 
       - name: 'Check translations output-file'
         working-directory: ${{ env.interfacePath }}
         run: |
           npm run set-env-variables -- env=development
           npm run extract-i18n
           git diff --exit-code src/locale/messages.xlf
+
+      - name: Save coverage report
```

Comment:
Why not directly AFTER the run of the `npm run test:coverage`-step?

Reply:
I figured that a failure in the "check translations" step is more likely than in the "save coverage report" step - so we'd like to make that happen sooner. And also the "save coverage report" step feels like an "end of the workflow" kind of thing (like the saving of artifacts). But I'm happy to move it if you think I should.

Reply:
Yeah, I had the same idea about the "end-of-the-run"ness of the step...
But then I thought that "waiting till the end" wouldn't make sense if it was marked `always()` anyway..

But now I think I don't really mind either way. :D